pkg/config/proxy.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+import (
+	"os"
+)
+
+const (
+	httpsProxyKey = "HTTPS_PROXY"
+	httpProxyKey  = "HTTP_PROXY"
+	noProxyKey    = "NO_PROXY"
+)
+
+func GetProxyConfigFromEnv() (map[string]string, error) {
```

Comment:
any reason why this needs to return an error?

Reply:
Nope, removed it 👍